### ISSUE TITLE
Public hosted dist file with a UMD

### ISSUE THREAD

**Issue by stefanjudis** [Yoast/YoastSEO.js#1192]
## What did you expect to happen?

I'd love to see a dist file e.g. on https://unpkg.com wrapped in a UMD.

## Technical info

I started playing around with YoastSEO.js and it works fine. The goal is to integrate Yoast in our API first CMS interface (we call this a UI-extension, that every user can upload). Unfortunately, we've got limitations of uploadable file sizes and YoastSEO is rather big. If i could load it somewhere from a CDN that would help me massively.

**Comment by atimmer** [comment]
Yes, totally something we want to do.

**Comment by stefanjudis** [comment]
Are you up for a PR solving this? I'd need this for a kinda "marketing" story. :D

**Comment by atimmer** [comment]
Yes, feel free to create a PR. In the future I would like to move to Webpack, but for now you can also implement it in browserify.
